homeassistant/components/remote/services.yaml

```diff
@@ -47,7 +47,7 @@ send_command:
       required: true
       example: "Play"
       selector:
-        text:
+        object:
```

Comment:
This is not correct. It's `vol.All(cv.ensure_list, [cv.string])`.

This would be better suited with a text with `multiple: true` (which we do not support right now)

Reply:
Uh, I'm not following?

This would allow bot string or a list to be accepted? We use this tactic in quite a few places?

Reply:
this would accept `x,x,x`, `["x", "x", "x"]` and 

```
- "x"
- "x"
- "x"
```

Reply:
It also allows to specify an incorrect data format like dictionaries. But if we use it in multiple places, then let's do it like this for now. Hopefully more selectors soon 👍